src/renderer/components/database-diagram.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { Component, PropTypes } from 'react';
+import joint from 'jointjs';
+
+require('./jointjs-diagram-table.js');
+require('./jointjs-diagram-table-cell.js');
+require('jointjs/dist/joint.min.css');
```

Comment:
I'll move styles specific to database diagram to separate .css file and load them after external styles.